plugin/main/src/kotlinx/benchmark/gradle/JsMultiplatformTasks.kt

```diff
@@ -73,11 +73,10 @@ private fun Project.createJsBenchmarkGenerateSourceTask(
         group = BenchmarksPlugin.BENCHMARKS_TASK_GROUP
         description = "Generate JS source files for '${target.name}'"
         title = target.name
-        ir = target.compilation is KotlinJsIrCompilation
         useBenchmarkJs = target.jsBenchmarksExecutor == JsBenchmarksExecutor.BenchmarkJs
         inputClassesDirs = compilationOutput.output.allOutputs
         inputDependencies = compilationOutput.compileDependencyFiles
         outputResourcesDir = file("$benchmarkBuildDir/resources")
         outputSourcesDir = file("$benchmarkBuildDir/sources")
     }
-}
+}
```

Comment:
Please try to avoid unnecessary changes like this.

Reply:
Noted.